executor/runcexecutor/executor.go

```diff
@@ -173,6 +177,10 @@ func (w *runcExecutor) Exec(ctx context.Context, meta executor.Meta, root cache.
 	if meta.ReadonlyRootFS {
 		opts = append(opts, containerdoci.WithRootFSReadonly())
 	}
+
+	if w.cgroupParent != "" {
+		opts = append(opts, containerdoci.WithCgroup(w.cgroupParent))
```

Comment:
This value should need the container id as well

Reply:
actually shouldn't it be `filepath.Join("/", w.cgroupParent, id)` ?